### PR TITLE
1.0 release

### PR DESCRIPTION
**Pull request by ichim-david**

### CODE REVIEW

**Review by avoinea — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -139,36 +122,32 @@
...
     "@storybook/react": "6.3.13",
-    "@whitespace/storybook-addon-html": "4.2.0",
     "eslint-plugin-prettier": "3.1.3",
     "jest-junit": "8.0.0",
-    "mrs-developer": "*",
-    "postcss": "8.3.11",
+    "mrs-developer": "^1.7.1",
+    "postcss": "8.4.5",
     "prettier": "2.0.5",
-    "razzle": "4.2.17",
```
Comment (valentinab25) [inline package.json:171]: Why did you remove the razzle dependency ?
